lib/db_dump_helper.rb

```diff
@@ -46,7 +46,7 @@ def self.dump_developer_db
   end
 
   def self.public_s3_path(file_name)
-    "https://s3.#{EnvConfig.STORAGE_AWS_REGION}.amazonaws.com/#{BUCKET_NAME}/#{file_name}"
+    "#{EnvConfig.ASSET_HOST}/#{file_name}"
```

Comment:
Why are you moving the database dumps to the assets host?

Reply:
Because we pay 300 bucks a month serving the database dumps directly from s3

Reply:
They are already in the asset bucket